packages/cli/src/commands/report.ts

```diff
@@ -74,8 +73,15 @@ export default class Report extends Command {
 
     absOutputPath = resolve(join(tbResultsFolder + `/${outputFileName}.pdf`));
 
-    chromeArgs = `--headless --disable-gpu --print-to-pdf file://${absPathToHTML}`;
-    execSync(`${findChrome()} ${chromeArgs}`);
+    const chromeExecutablePath = findChrome();
+
+    await execa(chromeExecutablePath, [
+      '--headless',
+      '--disable-gpu',
+      `--print-to-pdf=${absOutputPath}`,
+      `file://${absPathToHTML}`,
```

Comment:
please use https://nodejs.org/api/url.html#url_url_pathtofileurl_path instead of `file://${absPathToHTML}`

Reply:
Added fix in latest commit